packages/core/forms/src/components/fields/FieldArrayItem.vue

```diff
@@ -1,13 +1,18 @@
 <template>
-  <div class="array-item">
-    <slot />
-    <KButton
-      appearance="tertiary"
-      class="delete"
-      @click="$emit('remove-item')"
-    >
-      <TrashIcon />
-    </KButton>
+  <div class="array-item-wrapper">
```

Comment:
Is this new wrapper necessary? I don't know if this will break any style since the DOM structure is changed.

Reply:
Good catch. I feel it is redundant after some experiments. I've removed it.